samples/client/petstore/cpp-qt5/client/PFXHttpRequest.cpp

```diff
@@ -205,13 +189,12 @@ void PFXHttpRequestWorker::execute(PFXHttpRequestInput *input) {
                 request_content = "";
             }
         }
-    }
-    else {
+    } else {
         // variable layout is MULTIPART
 
-        boundary = "__-----------------------"
-            + QString::number(QDateTime::currentDateTime().toTime_t())
-            + QString::number(qrand());
+        boundary = QString("__-----------------------%0%1")
+                       .arg(QDateTime::currentDateTime().toTime_t())
+                       .arg(qrand());
```

Comment:
@etherealjoy switched to `.arg` with `%0%1` instead. Does it look good to you? I"ve not tested it myself.

From the doc (https://doc.qt.io/qt-5/qstring.html), looks like it usually starts with %1 instead of %0?

cc @MartinDelille

Reply:
It should be %1 %2, not from %0 indeed.

Reply:
Indeed my mistake. The thing is that on my side it works but the documentation say we should start from %1: https://doc.qt.io/qt-5/qstring.html#arg